packages/flutter/lib/src/widgets/app.dart

```diff
@@ -1197,7 +1198,29 @@ class WidgetsApp extends StatefulWidget {
   /// and view what widgets and render objects associated with it. An outline of
   /// the selected widget and some summary information is shown on device and
   /// more detailed information is shown in the IDE or DevTools.
-  static bool debugShowWidgetInspectorOverride = false;
+  @Deprecated(
+    'Use debugShowWidgetInspectorOverrideNotifier.value instead. '
+    'This feature was deprecated after v3.13.0-19.0.pre.'
+  )
+  bool get debugShowWidgetInspectorOverride =>
+      debugShowWidgetInspectorOverrideNotifier.value;
+  @Deprecated(
+    'Use debugShowWidgetInspectorOverrideNotifier.value instead. '
+    'This feature was deprecated after v3.13.0-19.0.pre.'
+  )
+  set debugShowWidgetInspectorOverride(bool value) {
+    debugShowWidgetInspectorOverrideNotifier.value = value;
+  }
+
+  /// If true, the WidgetInspector is added to the tree and made visible.
+  ///
+  /// Used by the `debugShowWidgetInspector` debugging extension.
+  ///
+  /// The inspector allows you to select a location on your device or emulator
+  /// and view what widgets and render objects associated with it. An outline of
+  /// the selected widget and some summary information is shown on device and
+  /// more detailed information is shown in the IDE or DevTools.
+  static ValueNotifier<bool> debugShowWidgetInspectorOverrideNotifier = ValueNotifier<bool>(false);
```

Comment:
This field should likely be `final`? We don't want people to replace the ValueNotifier, right?